packages/controls/src/widget_selectioncontainer.ts

```diff
@@ -209,7 +209,7 @@ export class AccordionView extends DOMWidgetView {
   }
 
   children_views: ViewList<DOMWidgetView> | null;
-  pWidget: Accordion;
+  lmWidget: Accordion;
```

Comment:
I think we also need `pWidget: Accordion` until it is removed (since it is right now inherited as type `Widget`)

Reply:
Unless there is some way to say that `pWidget` returns `typeof this.lmWidget` - I don't know if that will work with subclassing and redefining `lmWidget`, though.

Reply:
I didn't find a way to type `pWidget` in the base class as the type of `lmWidget` in the subclass, so I added back `pWidget`'s typing in each subclass as you initially suggested. Thanks!